master/buildbot/changes/gitpoller.py

```diff
@@ -159,8 +170,16 @@ def poll(self):
             '+%s:%s' % (self._removeHeads(branch), self._trackerBranch(branch))
             for branch in branches
         ]
-        yield self._dovccmd('fetch',
```

Comment:
Well, I tried using a non-existent change source and the error message was being logged correctly. But this does seem better to me as well. Will do.